src/Result/src/ResultFactory.cpp

```diff
@@ -0,0 +1,33 @@
+////////////////////////////////////////////////////////////////////////////////
+/**
+* @file ResultFactory.cpp
+* @brief Function definitions for Result Factory
+*/
+////////////////////////////////////////////////////////////////////////////////
+
+#include "Result.h"
+#include "ResultFactory.h"
+#include <sstream>
+#include <iostream>
+#include "CalculatorMessages.h"
+
+
+namespace calculator
+{
+
+//*************/
+// ResultFactory public methods /
+///
+   
+    Result ResultFactory::CreateResult(const Expression &parsedExpression, const float &answer)
+    {
+        Result result;
```

Comment:
As per comments in Result.h -- instead of directly modifying members of Result create a constructio and have it set members there